crates/ibc/src/core/ics04_channel/handler.rs

```diff
@@ -28,7 +16,6 @@ pub mod recv_packet;
 pub mod send_packet;
 pub mod timeout;
 pub mod timeout_on_close;
-pub mod write_acknowledgement;
 
 /// Defines the possible states of a channel identifier in a `ChannelResult`.
 #[derive(Clone, Debug)]
```

Comment:
Same here, the `ChannelResult` and the `ChannelIdState` are no longer used.

Reply:
[b9716d0](https://github.com/cosmos/ibc-rs/pull/456/commits/b9716d027ed8814128d93eb2d98733e53d52367b)